CONTRIBUTING.md

```diff
@@ -151,6 +151,11 @@ The release process is also automated in the way any specific commit from the ma
 1. Click the green `Run workflow` button to trigger the release workflow.
 1. Then you can go to the `https://www.npmjs.com/package/@elastic/synthetics` to validate that the bundles have been published.
 
+#### Versioning
```

Comment:
How does it pick the existing versioning if we set it to `0-0-0` in the package JSON, are we doing it based on existing tags? 

Also can you post a dry-run output? I am not able to run it localy.

Reply:
Yes it's base on existing tags.

Will post the log output in a couple of minutes. I will also add a how to test section in the description.